packages/@sanity/portable-text-editor/test/__tests__/writingTogether.collaborative.test.ts

```diff
@@ -117,22 +117,49 @@ describe('collaborate editing', () => {
       ]
     `)
     const selA = await editorA.getSelection()
-    expect(selA).toMatchInlineSnapshot(`null`)
```

Comment:
This is now changed because we set the selection again when a editor becomes empty and the user had a selection from before.

Also, different keys here as we have skipped one key generation call by removing the insert placeholder code from  `editor.onChange`.